openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -521,13 +510,50 @@ def process(self, instance):
                 "FTRACK_API_KEY": os.environ.get("FTRACK_API_KEY"),
                 "FTRACK_SERVER": os.environ.get("FTRACK_SERVER"),
             }
-            publish_job.update({"ftrack": ftrack})
+            publish_job["ftrack"] = ftrack
 
         metadata_path, rootless_metadata_path = \
-            create_metadata_path(instance, anatomy)
+                create_metadata_path(instance, anatomy)
+
+        with open(metadata_path, "w") as f_:
+            json.dump(publish_job, f_, indent=4, sort_keys=True)
+
+    def _create_default_render_job(self, instance):
+        import getpass
+
+        self.log.info("Faking job data ...")
```

Comment:
This should not be an info log. Also, I feel like it's lacking information about use cases where this might happen in the code. When do we ever fake it?